op-batcher/batcher/driver.go

```diff
@@ -273,6 +273,14 @@ func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context, start, end uin
 			return err
 		}
 		latestBlock = block
+
+		if (i-start)%100 == 0 {
+			// Every 100 blocks, signal the publishing loop to publish.
+			// This allows the batcher to start publishing sooner in the
+			// case of a large backlog of blocks to load.
+			trySignal(publishSignal)
```

Comment:
Perhaps we should also signal the throttling loop here, since we don't want to delay that work either in the case of a backlog.